src/attributes/classes.js

```diff
@@ -78,7 +76,7 @@ jQuery.fn.extend({
 
 				if ( cur ) {
 					j = 0;
-					while ( (clazz = classes[j++]) ) {
+					while ( ( clazz = classes[ j++ ] ) ) {
```

Comment:
The idea of the extra parens is to give the reader another visual clue that the single `=` isn't a mistaken `==` or `===`. [JSHint complains](http://jshint.com/docs/options/#boss) if you don't add them. Since the extra parens are not required, the minifier removes them.